libs/@blockprotocol/graph/src/graph-block-handler.ts

```diff
@@ -54,13 +54,17 @@ export class GraphBlockHandler
     element,
   }: {
     callbacks?: Partial<BlockGraphMessageCallbacks>;
-    element: HTMLElement;
+    element?: HTMLElement | null;
   }) {
     super({ element, serviceName: "graph", sourceType: "block" });
+
     if (callbacks) {
       this.registerCallbacks(callbacks);
     }
-    this.coreHandler.initialize();
+
+    if (element) {
+      this.initialize(element);
+    }
```

Comment:
I wonder if we could move these steps into the parent `ServiceHandler` class's constructor – if it accepted callbacks in the constructor, we wouldn't need to add these same steps in all its implementations (which we'll have to do for every new service if we keep it here). We could keep this constructor for better typing of the callbacks.

Reply:
I'm now calling initialize and registerCallbacks in the super constructor